### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by jhenshall**
Done some final bits to clean everything up and hopefully finalise the new workflow. Definitely need a check on whether these are the right things to do. 

Changes include:

- Update readme to reflect new workflow
- Delete `/docs` directory from `master` - this is no longer needed as site served from `gh-pages`
- Add `/docs` to `.gitignore` - allows a local dev environment without commiting it
- Remove .ideas folder (#16) Not updated for 3 years and not needed for site. @Hagellach37 - would be great if you could confirm this won't inadvertently break anything
- Remove folder and CNAME from old website setup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
 # MapSwipe Website
-This is the repository for the MapSwipe's website: http://mapswipe.org/
+This is the repository for the MapSwipe website: http://mapswipe.org/
```
Comment (laurentS) [inline README.md]: Can you link to http**S**://... instead? In 2020 we should not use unencrypted web anymore :)

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
 node_modules*
 .DS_Store
-docs/CNAME
\ No newline at end of file
+docs*
```
Comment (laurentS) [inline .gitignore]: I think this is incorrect. `docs*` will ignore any file whose name starts with `docs` such as `docstring` (which I don't think you want) but I'm not sure it covers files under the `docs/` folder. Looking at the example in the app's repo, I think `docs/` would be what you want. (and likewise 2 lines above: `node_modules/`

**Review by laurentS — APPROVED** [review]
that's quite the cleanup you've done there!  I just added a couple of nitpicks, as usual, but otherwise think it's great.

File: README.md (hunk 1 above)
Comment (jhenshall) [reply README.md]: Nice spot! Done.

File: .gitignore (hunk 2 above)
Comment (jhenshall) [reply .gitignore]: Yeah, I was unsure about this but copied what existed for `node_modules`. This makes sense. Done!

**Comment by Hagellach37** [comment]
looks good.

**Review by Hagellach37 — APPROVED** [review]

**Pull request merged**